File: java/2.3 Algoritmos de Búsqueda/2.3.1 Busqueda secuencial/BusquedaSecuencialTest.java
class BusquedaSecuencialTest {

	static int busquedaSecuencial(int datos[], int longitud,  int datoBuscado) {	
		for (int indice = 0; indice < longitud; indice ++)	{
			if (datos[indice] == datoBuscado) {
				return indice;
			}
		}
		return -1;
	}

	public static void main(String args[]) {
		int datos[] = { 8, 5, 3, 9, 1, 4, 7 };
		int longitud = datos.length;
		int datoBuscado = 9;
		
		int indice = busquedaSecuencial(datos, longitud, datoBuscado);
		System.out.print("El dato " + datoBuscado);
		if (indice != -1) {
			System.out.println(" está en el índice: " + indice);
		} else {
			System.out.println(" no se encontró");
		}
	}
	
	// Salida en consola: 
	// El dato 9 está en el índice: 3
}

File: python/2.3 Algoritmos de Búsqueda/secuencial/BusquedaSecuencialTest.py
#coding=utf-8

def busquedaSecuencial(datos, longitud,  datoBuscado):
	for indice in range(0,longitud):
		if (datos[indice] == datoBuscado): 
			return indice
	return -1

# Función principal 
if __name__ == "__main__":
	datos = [8, 5, 3, 9, 1, 4, 7]
	longitud = len(datos) 
	datoBuscado = 9
	indice = busquedaSecuencial(datos, longitud, datoBuscado)
	
	if (indice != -1):
		print(f"El dato {datoBuscado} está en el índice: {indice}")
	else:
		print(f"El dato {datoBuscado} no se encontró")

# Salida en consola: 
# El dato 9 está en el índice: 3